app/models/storage_location.rb

```diff
@@ -91,6 +91,14 @@ def inventory_total_value_in_dollars(inventory = nil)
     inventory&.total_value_in_dollars(storage_location: id)
   end
 
+  def inventory_total_quantity(inventory = nil)
+    if inventory
+      inventory.quantity_for(storage_location: id)
+    else
+      size
+    end
+  end
```

Comment:
Extracted this conditional to a method in the model to prevent duplication between the ERB files - do we like this approach?

Reply:
I'd rather just add a parameter to `size`where you can pass the inventory in to avoid reloading it. I.e.

```ruby
def size(inventory=nil)
  inventory ||= View::Inventory.new(organization_id)
  inventory.quantity_for(storage_location: id)
end
```

Reply:
This logic was refactored after the PR was opened, so this method was no longer needed. I've deleted it now